pkg/plugin/openai.go

```diff
@@ -22,7 +23,7 @@ func (a *App) newAuthenticatedOpenAIRequest(ctx context.Context, method string,
 	case openAIProviderAzure:
 		req.Header.Set("api-key", a.settings.OpenAI.apiKey)
 	case openAIProviderGrafana:
-		req.Header.Add("Authorization", "Bearer "+a.settings.LLMGateway.apiKey)
+		req.Header.Add("Authorization", "Basic "+a.settings.LLMGateway.apiKey)
```

Comment:
Ugh, this does seem to be the case, but is also really confusing as we don't think of api keys as already having the tenant id combined with it. Could we either rename the apiKey field or separate them out to just be an api key and then use
`fmt.Sprintf("Bearer %s:%s", a.settings.Tenant, a.settings.LLMGateway.apiKey)`
or
`req.SetBasicAuth(a.settings.Tenant, a.settings.LLMGateway.apiKey)`?

Reply:
Updated to use `SetBasicAuth` where applicable